front/app/utils/paginationUtils.ts

```diff
@@ -3,8 +3,8 @@ import { isNil } from 'lodash-es';
 
 export type SortDirection = 'ascending' | 'descending';
 
-export function getPageNumberFromUrl(url) {
-  if (!url || typeof url !== 'string' || url === '') return null;
```

Comment:
`!url` is already covered by `url === ''`. Any string that is not `'' `will return `false` with both `!url` and `url === ''`.